learn/advanced/pagination.md

```diff
@@ -0,0 +1,130 @@
+# Pagination of search results
+
+In a perfect world, users would not need to look beyond the first search result to find what they were looking for. In practice, however, it is usually necessary to create some kind of pagination interface to browse through long lists of results.
+
+In this guide, we will discuss some of Meilisearch's current limitations, how these limitations impact common pagination interface patterns, and the recommended way of handling pagination when using Meilisearch.
+
+## Choosing the right pagination UI
+
+There are quite a few pagination interfaces you might want to implement in your application. Many common UI patterns have a page selector allowing users to jump to any search results page. To create a page selector, you must know the exact number of total results so you can calculate the precise number of results pages.
+
+For performance reasons, however, Meilisearch cannot provide the exact number of results for a query. Instead, When using the search endpoint, responses contain an `estimatedTotalHits` field. As its name indicates, `estimatedTotalHits` is only an estimate of how many documents match your user's query.
+
+Because of this, we do not recommend creating interfaces with page selectors. If page selection is crucial to the software you are developing, see the [last section of this page](#not-recommended-page-selection) for tips that might help you work around Meilisearch's current limitations.
+
+We recommend creating pagination interfaces centered around [previous and next buttons](#recommended-previous-and-next-buttons).
+
+## Recommended: Previous and next buttons
+
+Using previous and next buttons for pagination means that users can easily page through results, but don't have the ability to jump to an arbitrary results page.
+
+![Placeholder image for prev/next interface]()
+
+Though this approach offers less precision than a full-blown page selector, it does not require knowing the precise number of search results. This makes it a good fit for Meilisearch's current capabilities.
```

Comment:
Could we add a small paragraph that informs the user that this pagination is very similar to the `infiniteScroll` and `load more` pagination type? Without going in deep explaination! Just to make the user aware of what they can do with meili and also make them aware of sexy pagination types that might inspire them not to use the classic finite pagination system :') 

If you prefer not, no problem!